pulp_ansible/app/tasks/collections.py

```diff
@@ -430,19 +430,54 @@ def _update_highest_version(collection_version):
     equals False on the last highest version and True on this version.
     Otherwise does nothing.
     """
+
+    def is_higher(a, b):
+        """Is a > b ?"""
+        if a.prerelease and b.prerelease and a > b:
+            return True
+        if a.prerelease and not b.prerelease:
+            return False
+        return a > b
```

Comment:
```suggestion
        if a.prerelease == b.prerelease
            return a > b
        return b.prerelease
```
I think this should coverall cases.
Also, i don't see the function being used (yet).

Reply:
Your function doesn't pass this test:

```
    def test_prerelease_not_higher_than_stable(self):
        namespace = randstr()
        name = randstr()
        specs = [
            (namespace, name, "1.0.1"),
            (namespace, name, "2.0.0-rc1"),
        ]
    
        collection_versions = build_cvs_from_specs(specs)
        for cv in collection_versions:
            _update_highest_version(cv, save=True)
    
>       assert (
            CollectionVersion.objects.filter(namespace=namespace, name=name, version="1.0.1")
            .first()
            .is_highest
            is True
        )
```